modules/prompt/themes/magicmace.zsh-theme

```diff
@@ -0,0 +1,95 @@
+#
+# magicmace theme
+# Ideas and code taken from:
+#   xero's zsh prompt <http://code.xero.nu/dotfiles>
+#   eriner's eriner prompt <https://github.com/Eriner/zim/blob/master/modules/prompt/themes/eriner.zsh-theme>
+#
+
+# Global variables
+function {
+  COLOR_ROOT="%F{red}"
+  COLOR_USER="%F{cyan}"
+  COLOR_NORMAL="%F{white}"
+  COLOR_ERROR="%F{red}"
+
+  if [[ "$EUID" -ne "0" ]]; then
+    USER_LEVEL="${COLOR_USER}"
+  else
+    USER_LEVEL="${COLOR_ROOT}"
+  fi
+}
+
+# Status:
+# - was there an error?
+# - are there background jobs?
+# - are we in a ranger session?
+prompt_status() {
+  local symbols=""
+
+  [[ ${RETVAL} -ne 0 ]]          && symbols+='${COLOR_ERROR}${RETVAL}${COLOR_NORMAL}'  # 'e' for error.
+  [[ $(jobs -l | wc -l) -gt 0 ]] && symbols+='b' # 'b' for background.
+  [[ ${RANGER_LEVEL} -ne 0 ]]    && symbols+='r' # 'r' for... you guessed it!
+
+  [[ -n ${symbols} ]] && print -Pn "─${COLOR_NORMAL}${symbols}${COLOR_USER}─"
+}
+
+prompt_git() {
+  local ico_dirty='*'
+  local ico_ahead='🠙'
+  local ico_behind='🠛'
+  local ico_diverged='⥮'
+  local git_info=""
+
+  is_dirty() {
+    test -n "$(git status --porcelain --ignore-submodules)"
```

Comment:
`$(command git status...` is probably what you want. Aliases can cause unexpected issues otherwise.